### ISSUE TITLE
qwertz supprt

### ISSUE THREAD

**Issue by Velrok** [jigish/slate#273]
Could you support the QWERTZ keyboard layout?

On the german layout the ö button is on the right side of the l button.
So ö is part of the home row specially the right pinky finger.

I would like to use this key in key bindings:

```
bind ö:${hyper} <some move command>
```

**Comment by deiga** [comment]
In what way? You can define the keys used for switcher yourself

**Comment by Velrok** [comment]
How can I do that?
Is switcher a 3rd party program?

**Comment by deiga** [comment]
when you bind hint you can give it a string of characters which it uses. You do this in the configuration

**Comment by Velrok** [comment]
Sorry I don't get you :( .

I want to do something like this:

```
bind 'ö':${hyper} ${right}
```

**Comment by deiga** [comment]
Ah, your issue topic is totally misleading.
What you want is unicode character support, it has nothing to do with the arrangement of your keys on the keyboard..

**Comment by Velrok** [comment]
Yes and no. I guess my key arrangement contains unicode characters :(, which I would like to use.

I also changed the description. Hopefully my intentions are clearer now.
@deiga Thanks for all the comments and suggestions.

**Comment by tadhg-ohiggins** [comment]
This is really a duplicate of https://github.com/jigish/slate/issues/133

Also, the workaround I suggest in the discussion of that issue should work for this.
